bindings/java/nz/mega/sdk/MegaApiJava.java

```diff
@@ -8764,99 +8875,266 @@ public long getBandwidthOverquotaDelay() {
+     *                     Sort by creation time in MEGA, descending
      *
-     *  - MegaApi::ORDER_MODIFICATION_ASC = 7
-     *  Sort by modification time of the original file, ascending
+     *                     - MegaApi::ORDER_MODIFICATION_ASC = 7
+     *                     Sort by modification time of the original file, ascending
      *
-     *  - MegaApi::ORDER_MODIFICATION_DESC = 8
-     *  Sort by modification time of the original file, descending
+     *                     - MegaApi::ORDER_MODIFICATION_DESC = 8
+     *                     Sort by modification time of the original file, descending
      *
-     *  - MegaApi::ORDER_ALPHABETICAL_ASC = 9
-     *  Sort in alphabetical order, ascending
+     *                     - MegaApi::ORDER_ALPHABETICAL_ASC = 9
+     *                     Same behavior than MegaApi::ORDER_DEFAULT_ASC
      *
-     *  - MegaApi::ORDER_ALPHABETICAL_DESC = 10
-     *  Sort in alphabetical order, descending
+     *                     - MegaApi::ORDER_ALPHABETICAL_DESC = 10
+     *                     Same behavior than MegaApi::ORDER_DEFAULT_DESC
      *
-     * @return List of nodes that contain the desired string in their name.
+     *                     Deprecated: MegaApi::ORDER_ALPHABETICAL_ASC and MegaApi::ORDER_ALPHABETICAL_DESC
+     *                     are equivalent to MegaApi::ORDER_DEFAULT_ASC and MegaApi::ORDER_DEFAULT_DESC.
+     *                     They will be eventually removed.
+     *
+     *                     - MegaApi::ORDER_PHOTO_ASC = 11
+     *                     Sort with photos first, then by date ascending
+     *
+     *                     - MegaApi::ORDER_PHOTO_DESC = 12
+     *                     Sort with photos first, then by date descending
+     *
+     *                     - MegaApi::ORDER_VIDEO_ASC = 13
+     *                     Sort with videos first, then by date ascending
+     *
+     *                     - MegaApi::ORDER_VIDEO_DESC = 14
+     *                     Sort with videos first, then by date descending
+     *
+     *                     - MegaApi::ORDER_LABEL_ASC = 17
+     *                     Sort by color label, ascending
+     *
+     *                     - MegaApi::ORDER_LABEL_DESC = 18
+     *                     Sort by color label, descending
+     *
+     *                     - MegaApi::ORDER_FAV_ASC = 19
+     *                     Sort nodes with favourite attr first
+     *
+     *                     - MegaApi::ORDER_FAV_DESC = 20
+     *                     Sort nodes with favourite attr last
+     *
+     * @return List of nodes that contain the desired string in their name
      */
-    public ArrayList<MegaNode> search(String searchString, int order) {
-        return nodeListToArray(megaApi.search(searchString, order));
+    public ArrayList<MegaNode> searchOnInShares(String searchString, MegaCancelToken cancelToken, int order) {
+        return nodeListToArray(megaApi.search(searchString, cancelToken, order));
```

Comment:
This binding should call to `MegaApi::searchOnInShares()`
```suggestion
        return nodeListToArray(megaApi.searchOnInShares(searchString, cancelToken, order));
```